src/styles/blocks/header.scss

```diff
@@ -0,0 +1,71 @@
+body {
+  margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
+  &__link {
+    color: #060b35;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    line-height: 60px;
+    height: 60px;
+    display: block;
+    transition: $transition-time;
```

Comment:
The variable `$transition-time` is used here. Ensure that this variable is defined in your SCSS files, possibly in a variables file, to avoid compilation errors.